### PR TITLE
Added langs

### PR DESCRIPTION
**Pull request by maxigator**
Added langs : 
de_de
en_gb
es_es
fr_fr
it_it
ja_jp
nl_nl
pr_pt
ru_ru

Let me know if you want me to change the way i translated

### CODE REVIEW

**Comment by Yales21** [comment]
There is a little mistake with the French. It is not "Shaders actualisé**e**s !" but ”Shaders actualisés !"

**Comment by maxigator** [comment]
Are you sure because I thought that "shaders" was feminine because its "ombres" which is feminine so idk

**Comment by maxigator** [comment]
Oh sorry I just looked it up and its un shader and not une shader

**Comment by Pepe20129** [comment]
The spanish translation is ok.

**Comment by wurunduk** [comment]
Russian translation is quite wrong, here is a fix:
`
{
  "iris.shaders.reloaded": "Шейдеры перезагружены!",  
  "iris.keybind.reload": "Перезагрузить шейдеры",  
  "iris.keybinds": "Iris",  
  "iris.shaders.reloaded.failure": "Ошибка при перезагрузке шейдеров: %s"  
} 
`

File: src/main/resources/assets/iris/lang/es_es.json
Comment (altrisi) [inline src/main/resources/assets/iris/lang/es_es.json:5]: I think the correct word in Spanish for "reload" would be "recargar", hence the following suggestion:
```suggestion
  "iris.shaders.reloaded": "Shaders recargados!",
  "iris.keybind.reload": "Recargar Shaders",
  "iris.keybinds": "Iris",
  "iris.shaders.reloaded.failure": "Error al recargar los shaders: %s"
```
I'd say "actualizar" is more like "update".

File: src/main/resources/assets/iris/lang/en_gb.json
Comment (coderbot16) [inline src/main/resources/assets/iris/lang/en_gb.json:1]: If you're not changing anything from the en_us file, then no need to have this to begin with.

File: src/main/resources/assets/iris/lang/de_de.json
Comment (coderbot16) [inline src/main/resources/assets/iris/lang/de_de.json:4]: Since you're not changing this, don't add it to the translation files because it will automatically fall back to en_us. It's only necessary to add entries for things that differ from en_us.
```suggestion
```

File: src/main/resources/assets/iris/lang/ru_ru.json
Comment (coderbot16) [inline src/main/resources/assets/iris/lang/ru_ru.json:5]: JustMarkov translated this slightly differently at https://github.com/JustMarkov/Iris/commit/d8dd925bad0a2b521beeefdd1a5155ce52dc1e9f :

```
  "iris.shaders.reloaded.failure": "Ошибка при Перезагрузке шейдеров! Причина: %s"
```

Not sure which is better, however I think JustMarkov's version is closer in form to the original English version.

However, it's possible that "Failed to reload shaders: %s" like what you did might be a better way to write things in the english version to... I'm split on this.

**Review by coderbot16 — CHANGES_REQUESTED** [review]
The translations seem reasonable enough, the only problem at the moment is that it looks like only fr_fr, es_es, and ru_ru have been verified though. I have no way to know whether the other translations are good or not, and I guess that just calls into question whether it's better to have translations that potentially have small errors or no translations at all. In any case, the translations seem to look fine to me otherwise.

Thanks for your contributions!

File: src/main/resources/assets/iris/lang/ru_ru.json
Comment (maxigator) [reply src/main/resources/assets/iris/lang/ru_ru.json]: I decided to change all error messages to the same as yours 
Like this "Failed to Reload Shaders! Reason: %s"

File: src/main/resources/assets/iris/lang/fr_fr.json
Comment (arthurbambou) [inline src/main/resources/assets/iris/lang/fr_fr.json:4]: Upper case e of Echec needs an accent
```suggestion
  "iris.shaders.reloaded.failure": "Échec lors du rechargement des shaders! Raison: %s"
```

**Comment by FoundationGames** [comment]
The GUI pull request is going to come in once the shader options pull request is merged, and the GUI PR has a lot of new lang entries that need to be translated... Either we can put the PR on hold or just add the new lang entries right now.
Here is the updated version of en_us.json in my fork (subject to change):
```json
{
  "iris.shaders.reloaded": "Reloaded Shaders!",
  "iris.keybind.reload": "Reload Shaders",
  "iris.keybinds": "Iris",
  "iris.shaders.reloaded.failure": "Failed to Reload Shaders! Reason: %s",

  "options.iris.apply": "Apply",
  "options.iris.openShaderPackFolder": "Open Shader Pack Folder...",
  "options.iris.refreshShaderPacks": "Refresh Shader Packs",
  "options.iris.shaderPackSelection": "Shader Packs...",
  "options.iris.shaderPackSelection.title": "Shader Packs",

  "option.iris.back": "Back",

  "pack.iris.select.title": "Select",
  "pack.iris.configure.title": "Configure",

  "page.iris.notFound": "Invalid Page!",
  "page.iris.noConfig": "No Shader Options.",

  "property.iris.boolean.true": "True",
  "property.iris.boolean.false": "False"
}
```

**Comment by coderbot16** [comment]
I would rather just do these translations now, then add those ones in a future PR

**Review by arthurbambou — APPROVED** [review]

**Comment by FoundationGames** [comment]
May be a bit inconvenient but I just added a few translations to my GUI, I updated my message above

**Comment by maxigator** [comment]
I believe that `shaders reloaded` is better that `reloaded shaders`

File: src/main/resources/assets/iris/lang/fr_fr.json
Comment (toinouH) [inline src/main/resources/assets/iris/lang/fr_fr.json:4]:
```suggestion
  "iris.shaders.reloaded.failure": "Échec lors du rechargement des shaders ! Raison: %s"
```
Same :)

**Review by toinouH — CHANGES_REQUESTED** [review]
Almost ready :)

**Review by toinouH — APPROVED** [review]
Perfect :D

**Comment by maxigator** [comment]
thanks for the help! in french I thought that there was a space before every "!".

**Review by coderbot16 — APPROVED** [review]
Looks good for now :+1:

**Pull request merged**
